### PR TITLE
Clarified notebook setup in Fairing guide

### PR DESCRIPTION
**Pull request by sarahmaddox**
In the Fairing guide to training/deploying from a Kubeflow notebook, this PR makes it clear that the Kubeflow notebook already includes Fairing as well as other ML libraries. This PR also links to our new guide to setting up notebooks.

fixes https://github.com/kubeflow/website/issues/644

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/645)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-645--competent-brattain-de2d6d.netlify.com/docs/fairing/gcp-kubeflow-notebook/

**Comment by sarahmaddox** [comment]
This PR is ready for review. Adding a hold for tech writer review pending

/hold

File: content/docs/fairing/gcp-kubeflow-notebook.md [hunk 1]
```diff
@@ -15,27 +32,15 @@ to demonstrate how to:
...
-    in Kubeflow. 
+Follow these instructions to run the XGBoost quickstart notebook:
 
 1.  Download the files used in this example and install the packages that the
     XGBoost quickstart notebook depends on.
 
-    1.  In your hosted notebook environment, click **New** and select **Terminal**
-        to start a new terminal session in your notebook environment. Use the
-        terminal session to set up your notebook environment to run this example.
+    1.  On the **Jupyter dashboard** for your notebook server, click **New** and
```
Comment (karthikv2k) [inline content/docs/fairing/gcp-kubeflow-notebook.md]: same comment for "notebook server"

File: content/docs/fairing/gcp-kubeflow-notebook.md [hunk 2]
```diff
@@ -6,7 +6,24 @@ weight = 35
 
 This guide introduces you to using [Kubeflow Fairing][fairing-repo] to train and
 deploy a model to Kubeflow on Google Kubernetes Engine (GKE) and Google Cloud
-ML Engine. As an example, this guide uses a notebook that is hosted on Kubeflow
+ML Engine.
```
[the remark this answers is not in the document]
Comment (sarahmaddox) [reply content/docs/fairing/gcp-kubeflow-notebook.md]: Done.

File: content/docs/fairing/gcp-kubeflow-notebook.md [hunk 3]
```diff
@@ -6,7 +6,24 @@ weight = 35
...
+
+Your Kubeflow deployment includes services for spawning and managing Jupyter
+notebooks. Kubeflow Fairing is preinstalled in the Kubeflow notebooks, along
+with a number of machine learning (ML) libraries.
+
+## Set up Kubeflow and access the Kubeflow notebook environment
+
+Follow the [Kubeflow notebook setup guide](/docs/notebooks/setup/)
+to install Kubeflow, access your Kubeflow hosted notebook environment, and 
+create a new notebook server.
```
[the remark this answers is not in the document]
Comment (sarahmaddox) [reply content/docs/fairing/gcp-kubeflow-notebook.md]: When the user accesses "Notebooks" within Kubeflow, the first thing they need to do is click **NEW SERVER** and add a server. See the [docs](https://www.kubeflow.org/docs/notebooks/setup/#create-a-jupyter-notebook-server-and-add-a-notebook). The image they select is to configure the server. Given the terminology on the Kubeflow UI, are you happy to keep the word "server" here?

File: content/docs/fairing/gcp-kubeflow-notebook.md [hunk 4]
```diff
@@ -6,7 +6,24 @@ weight = 35
...
+
+When selecting a Docker image and other settings for the baseline deployment
+of your notebook server, you can leave all the settings at the default value.
```
[the remark this answers is not in the document]
Comment (sarahmaddox) [reply content/docs/fairing/gcp-kubeflow-notebook.md]: Same reply as above.

File: content/docs/fairing/gcp-kubeflow-notebook.md (hunk 1 above)
Comment (sarahmaddox) [reply content/docs/fairing/gcp-kubeflow-notebook.md]: Same reply as above.

**Comment by sarahmaddox** [comment]
Thanks for your review @karthikv2k PTAL at my replies.

File: content/docs/fairing/gcp-kubeflow-notebook.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (karthikv2k) [reply content/docs/fairing/gcp-kubeflow-notebook.md]: Ok. then it makes to call server here.

**Comment by sarahmaddox** [comment]
Thanks @thedriftofwords - fixed.

**Pull request merged**